vars/snyk.groovy

```diff
@@ -14,7 +14,6 @@ def snykCodeTest(Map snykCodeTestArgs=[:]) {
 
 def snykTest(Map snykTestArgs=[:]) {
     logger.info("Snyk Test")
-    logger.debug("snykTestArgs: ${snykTestArgs}")
     logger.info("Technology: ${snykTestArgs.tech}")
     def xrayCommand = "snyk test --org=${snykTestArgs.snyk.orgId} --json"
```

Comment:
what does xrayCommand mean in this context?

Reply:
I had copied the variable name over from where I had done something similar in JFrog's file. Changing to snykTestCommand